.log/ti-56041.log

```diff
@@ -0,0 +1,139 @@
+[06:01:02.264] Global cache location '/Users/yosevu/Library/Caches/typescript/4.7', safe file path '/users/yosevu/.nvm/versions/node/v14.17.0/lib/node_modules/typescript/lib/typingsafelist.json', types map path /users/yosevu/.nvm/versions/node/v14.17.0/lib/node_modules/typescript/lib/typesmap.json
```

Comment:
Can you please remove these log-files from being added to the repo?

Reply:
Oops, I didn't realize I committed them.